code/modules/power/apc.dm

```diff
@@ -912,6 +912,8 @@
 	return 1
 
 /obj/machinery/power/apc/proc/toggle_breaker()
+	if((stat & (BROKEN|MAINT)) || failure_timer)
```

Comment:
use is_operational() instead of BROKEN, check and see if it check MAINT and failure_timer too